foerster-geisteswissenschaft.csl

```diff
@@ -27,23 +27,28 @@ INFORMATIONEN ZUM ZITIERSTIL
     <category field="generic-base"/>
     <summary>German generic-base style for humanities. Citations in notes; full bibliography.</summary>
     <published>2012-05-28T14:00:00+00:00</published>
-    <updated>2013-09-09T14:00:00+00:00</updated>
+    <updated>2015-02-14T08:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <!--
+    <!--
 ================================================================================
-TODO
+Changelog
 ================================================================================
--->
-  <!--
+ 
+2015-02-14: 
+- Unterstützung für die Typen "Dissertation" (thesis), "Dokument" (article) und "Manuskript" (manuscript)
+- Kurztitel wird verwendet, wenn vorhanden. Dadurch wird auch die Option zum Benutzen der Medline-Zeitschriftkürzel in den Pref Doc Settings eingeblendet. Wer den Langtitel nutzen möchte, muss die Kurztitel aus der Datenbank entfernen.
+- Zusätzlich zu Webseiten und Dissertation wird jetzt auch bei Magazin-, Zeitungs- und Journal-Artikel  die URL, wenn vorhanden, angezeigt.
+ 
+ 
 ================================================================================
 ÜBERSETZUNGEN
 ================================================================================
 -->
   <locale xml:lang="de">
     <terms>
-      <!-- Begriffe -->
-      <term name="accessed">abgerufen am</term>
+      <!-- besondere Begriffe -->
+      <term name="accessed">abgerufen am</term>     
       <term name="and">und</term>
```

Comment:
Oh, actually I had some problems with the local a long while ago. To have more control, I copied most used terms over here. Is there a file, where I can compare them and delete duplicates?